cdk/package.json

```diff
@@ -20,6 +20,7 @@
     "typescript": "~4.5.4"
   },
   "dependencies": {
+    "@aws-cdk/aws-lambda-event-sources": "^1.137.0",
```

Comment:
Das sieht komisch aus wenn der Rest CDK v2 ist...

Reply:
Thanks, we can get rid of that, was added by error